static/app/views/performance/newTraceDetails/traceOurlogs.tsx

```diff
@@ -0,0 +1,77 @@
+import type React from 'react';
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import {SearchQueryBuilder} from 'sentry/components/searchQueryBuilder';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {
+  LogsPageDataProvider,
+  useLogsPageData,
+} from 'sentry/views/explore/contexts/logs/logsPageData';
+import {
+  LogsPageParamsProvider,
+  useLogsSearch,
+  useSetLogsQuery,
+} from 'sentry/views/explore/contexts/logs/logsPageParams';
+import {LogsTable} from 'sentry/views/explore/logs/logsTable';
+import {SectionKey} from 'sentry/views/issueDetails/streamline/context';
+import {InterimSection} from 'sentry/views/issueDetails/streamline/interimSection';
+
+export type UseTraceViewLogsDataProps = {
+  children: React.ReactNode;
+  traceSlug: string;
+};
+
+export function TraceViewLogsDataProvider({
+  traceSlug,
+  children,
+}: UseTraceViewLogsDataProps) {
+  return (
+    <LogsPageParamsProvider isOnEmbeddedView limitToTraceId={traceSlug}>
+      <LogsPageDataProvider>{children}</LogsPageDataProvider>
+    </LogsPageParamsProvider>
+  );
+}
+
+export function TraceViewLogsSection() {
+  return (
+    <InterimSection
+      key="logs"
+      type={SectionKey.LOGS}
+      title={t('Logs')}
+      data-test-id="logs-data-section"
+      initialCollapse={false}
+    >
+      <LogsSectionContent />
+    </InterimSection>
+  );
```

Comment:
Do we want to show a foldable section with no content? I think maybe move the 
```
const tableData = useLogsPageData();
``` 
hook into this section then just re-use `LogsSectionContent` from LogIssueSection